src/carbonio-files-ui-common/views/components/ItemTypeChipInput.tsx

```diff
@@ -65,6 +65,7 @@ export const ItemTypeChipInput: React.VFC<ItemTypeChipInputProps> = ({
 }) => {
 	const [t] = useTranslation();
 	const [filterValue, setFilterValue] = useState<string | null>(null);
+	const [icon, setIcon] = useState('ChevronDown');
```

Comment:
Add the type <keyof DefaultTheme['icons']> to have a strictier check on the string